edge/pkg/metamanager/apiserver-lite/kubernetes/serializer/serializer.go

```diff
@@ -0,0 +1,53 @@
+package serializer
```

Comment:
If copy from k8s, need to add k8s copyright then add comment about changes.

Reply:
how about is just partly copied from apiserver and modified?

Reply:
Comment which file we copy and use below format log to mark changes:
```
@CHANGELOG
KubeEdge Authors: To xxx
Changes done are
1. xxx

```